packages/lisk-cryptography/src/convert.js

```diff
@@ -14,6 +14,7 @@
  */
 import querystring from 'querystring';
 import bignum from 'browserify-bignum';
+import reverse from 'buffer-reverse';
```

Comment:
I think we should be a bit more clear on function names, so I suggest to change this to be `bufferReverse`.